easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-8-foss-2020b.eb

```
@@ -0,0 +1,39 @@
+name = 'OpenFOAM'
+version = '8'
+
+homepage = 'https://www.openfoam.org/'
+description = """OpenFOAM is a free, open source CFD software package.
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer,
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/OpenFOAM/OpenFOAM-%(version_major)s/archive']
+sources = ['version-%(version)s.tar.gz']
+patches = [
+    'OpenFOAM-%(version)s-cleanup.patch',
+]
+checksums = [
+    '94ba11cbaaa12fbb5b356e01758df403ac8832d69da309a5d79f76f42eb008fc',  # version-8.tar.gz
+    'e3fac8049611ee109f07eb5b0ed4b1526782a21b3f12435e03529d56c8483b9f',  # OpenFOAM-8-cleanup.patch
+]
+
+dependencies = [
+    ('libreadline', '8.0'),
+    ('ncurses', '6.2'),
+    # OpenFOAM requires 64 bit METIS using 32 bit indexes (array indexes)
+    ('METIS', '5.1.0'),
+    ('SCOTCH', '6.1.0'),
+    ('CGAL', '5.2'),
+    ('ParaView', '5.8.1', '-mpi'),
+]
```

```suggestion
    ('gnuplot', '5.4.1'),
]
```

Add gnuplot, as per https://github.com/easybuilders/easybuild-easyblocks/pull/2256